changelog.txt

```diff
@@ -1,5 +1,6 @@
 v.Next
 ----------
+- [PATCH] Update common dependency to 3.2.0 (#1591)
```

Comment:
Seems there is already a line for down below on line 7....could just update there....but with that said, I prefer not documenting common change in this way because we don't really know whether it is MAJOR, MINOR or PATCH...or may be we do if we look at common's changelog, however, I think we usually document common dependency change as follows

- Picks up common@3.2.0

so it is neither of major, minor or patch

Reply:
thanks @shahzaibj  for the feedback :D, so in conclusion the preferred way to approach this is to remove line 7, and leave only
* Picks up common@3.2.0
right?

Reply:
Yes my recommendation would be to remove both line 3 and 7, and add a line for `Picks up common@3.2.0` - though this value will probably change whenever we actually plan to release another version of ADAL...so we can probably just leave this line out but doesn't harm to add it there either.

cc/ @iambmelt